src/network/server.rs

```diff
@@ -283,11 +284,13 @@ impl<N: Network, E: Environment> Server<N, E> {
     ) {
         if E::NODE_TYPE == NodeType::Miner {
             if let Some(recipient) = miner {
+                // TODO: add data_path as a constant string when #20 is merged.
+                let wallet = Wallet::new(&recipient.to_string(), &".aleo", false).expect("Should be able to open wallet database");
```

Comment:
```suggestion
                let wallet = Wallet::new(&recipient.to_string(), ".aleo", false).expect("Should be able to open wallet database");
```
also, is there a reason why `recipient` is converted to a `String` here? it would work with the later `format!` without conversion, and the `Wallet` could contain the original type instead